app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/SessionDataExtractor.scala

```diff
@@ -95,3 +95,20 @@ trait SessionDataExtractor extends Results {
     }
 
 }
+
+//This method should be used in controllers, where we did not introduce the JourneyBindable yet
+object TemporaryJourneyExtractor extends SessionDataExtractor {
```

Comment:
i know this is only temporary but i still think a dedicated unit test for all the cases would be a good idea.

Reply:
Done